### ISSUE TITLE
Investigate why cats and dogs example in keras has low accuracy 

### ISSUE THREAD

**Issue by sukanyaX** [srodney/diffimageml#127]
here are a few examples: 
https://towardsdatascience.com/step-by-step-vgg16-implementation-in-keras-for-beginners-a833c686ae6c -- this one is closest to the implementation in Duev et al. 2019 so I mainly played around with this.  You can vary the learning rate (lr) or the optimizer, though Duev et al used the Adam optimizer

This is another example which is similar but with the momentum optimizer: 
https://machinelearningmastery.com/how-to-develop-a-convolutional-neural-network-to-classify-photos-of-dogs-and-cats/

You can get the data here: https://www.kaggle.com/c/dogs-vs-cats/data
